test_xpartition.py

```diff
@@ -218,12 +213,22 @@ def func(ds):
     )
     if has_coord:
         ds = ds.assign_coords(x=range(5))
+    if has_chunked_coord:
+        chunked_coord = xr.DataArray(range(5), dims=["x"]).chunk({"x": 5})
+        ds = ds.assign_coords(b=chunked_coord)
+
+    unchunked_variables = xpartition.get_unchunked_variable_names(ds)
 
     store = str(tmpdir)
     mapper = ds.z.partition.map(store, ranks=3, dims=["x"], func=func, data=ds)
-    for rank in mapper:
+    for i, rank in enumerate(mapper):
+        if i == 0:
+            expected_times = checkpoint_modification_times(store, unchunked_variables)
```

Comment:
Initialization of the store only happens when we first start iterating through the mapper (see [here](https://github.com/spencerkclark/xpartition/blob/9e319e72cef7a5f488846057b932c2bbd10a277f/xpartition.py#L579)).